docs/guides/basics/parameters/index.md

```diff
@@ -131,10 +146,30 @@ Images and files that the user uploaded to the doc will be hosted on the `codaho
 
 If you need access to the binary content of the image or file you'll need to use the [fetcher][fetcher_binary_response] to retrieve it. The fetcher is automatically allowed access to the `codahosted.io` domain, so no need to declare it as a [network domain][network_domains]. It's not possible to access the binary content of images coming from an **Image URL** column, since they can come from any domain.
 
+??? example "Example: Image file size formula"
+    ```ts
+    --8<-- "examples/image/file_size.ts"
+    ```
+
 
 ### Lists
 
-Each of the parameter types described above has an array variant that allows you to pass a list of values of that type. For example, `StringArray` and `NumberArray`. All of the values in the list must be of the same type.
+Each of the parameter types described above has an array variant that allows you to pass a list of values of that type. For example, `StringArray` and `NumberArray`. All of the values in the list must be of the same type and not blank.
+
+??? example "Example: Longest string formula"
+    ```ts
+    --8<-- "examples/parameter/longest.ts"
+    ```
+
+
+### Table column
+
+Passing a table column into an array parameter can be error prone, because if that column is empty for some rows then the formula will fail to run. To accept a list of values that may include empty values there use the sparse variant of the array parameter (`SparseStringArray`, `SparseNumberArray`, etc). Blank cells will be representd as `null`, and you'll need to make sure your code can handle those values.
```

Comment:
maybe `there use` -> `there, use`? ignore if it's just my reading..

Reply:
Good catch, some leftover word. Removed, and made other fixes / improvements.